pandas/tests/test_groupby.py

```diff
@@ -5986,11 +5986,44 @@ def test_groupby_whitelist(self):
             # 'nlargest', 'nsmallest',
         ])
 
+        # TODO: Fix these inconsistencies between attribute and method names
+        inconsistently_named = frozenset([
+            'tshift',
+            'any',
+            'dtypes',
+            'idxmax',
+            'all',
+            'fillna',
+            'rank',
+            'quantile',
+            'cummax',
+            'take',
+            'corr',
+            'cummin',
+            'diff',
+            'plot',
+            'pct_change',
+            'skew',
+            'hist',
+            'bfill',
+            'cov',
+            'boxplot',
+            'describe',
+            'corrwith',
+            'idxmin',
+            'ffill',
+            'mad',
+            'dtype',
+            'unique'
+        ])
+
         for obj, whitelist in zip((df, s), (df_whitelist, s_whitelist)):
```

Comment:
what I mean is create *another* function where you assert that these are NOT equal (with the sub-list of those that are broken), so that when they are fixed this tests will break (and the list will need to be updated). IOW, create a list like the white-list of the broken ones, have the existing test just skip on them (as you do below), but then create *another* function that asserts that the broken ones are not equal.